tests/95-functions-sd.js

```diff
@@ -102,23 +110,26 @@ describe('Functions - ecdsa-sd-2023', function() {
         'PROOF_VERIFICATION_ERROR.',
       async function() {
         this.test.link = 'https://www.w3.org/TR/vc-di-ecdsa/#selective-disclosure-functions';
-        const proof = proofExists(securedCredential);
-        should.exist(proof.proofValue,
-          'Expected proof to have proofValue.');
-
-        // Create negative fixture
-        const invalidBaseCredential = structuredClone(securedCredential);
-        invalidBaseCredential.proof.proofValue =
-          invalidBaseCredential.proof.proofValue.slice(1);
-        ({disclosedCredential} = await createDisclosedVc(
-          {
-            selectivePointers: ['/credentialSubject/id'],
-            signedCredential: invalidBaseCredential,
-            vcHolder: holder
-          }));
-        should.not.exist(disclosedCredential?.proof,
-          'Derive endpoint should reject proof without multibase indicator.'
-        );
+        if(holder) {
+          const proof = proofExists(securedCredential);
+          should.exist(proof.proofValue,
+            'Expected proof to have proofValue.');
+          // Create negative fixture
+          const invalidBaseCredential = structuredClone(securedCredential);
+          invalidBaseCredential.proof.proofValue =
+            invalidBaseCredential.proof.proofValue.slice(1);
+          ({disclosedCredential} = await createDisclosedVc(
+            {
+              selectivePointers: ['/credentialSubject/id'],
+              signedCredential: invalidBaseCredential,
+              vcHolder: holder
+            }));
+          should.not.exist(disclosedCredential?.proof,
+            'Derived endpoint should reject proof without multibase indicator.'
```

Comment:
This is the `/derive` endpoint:

```suggestion
            'Derive endpoint should reject proof without multibase indicator.'
```